src/_ext.cpp

```diff
@@ -1617,17 +1618,12 @@ PYBIND11_MODULE(_ext, m) {
           auto prev = ptrjetoffsets[jetidx-1];
 
           for (unsigned int j = 0; j < jets.size(); j++){
-            // adapted from https://github.com/fdreyer/LundPlane/blob/master/LundGenerator.cc
-            PseudoJet pair, j1, j2;
-            pair = jets[j];
-            int splittings = 0;
-            while (pair.has_parents(j1, j2)) {
-              if (j1.pt2() < j2.pt2()) std::swap(j1,j2);
-              double Delta = j1.delta_R(j2);
-              Delta_vec.push_back(Delta);
-              kt_vec.push_back(j2.pt() * Delta);
-              pair = j1;
-              splittings++;
+            auto lund_generator = fastjet::contrib::LundGenerator();
```

Comment:
Would it be possible to instantiate this outside of the double-nested loop?

Reply:
oops